base/sysimg.jl

```diff
@@ -23,8 +23,8 @@ let
     # Run with the `--exclude-jlls` option to filter out all JLL packages
     stdlibs = [
         # No dependencies
-        :ArgTools,
         :Artifacts,
+        :ArgTools,
```

Comment:
Out of curiosity, why reorder? (You successfully nerd sniped me 😄.)

Reply:
oops, that wasn't for this PR (I was timing ArgTools, and wondering why it was one of the slower packages to load)